templates/Default/base.html.twig

```diff
@@ -44,6 +44,7 @@
 {% include 'Default/footer.html.twig' %}
 
 <div id="app-version" data-app-version="{{ app_version }}" style="display: none;">{{ app_version }}</div>
+<div id="app-language" data-app-language="{{ app.request.locale }}"></div>
```

Comment:
I could have changed this to remove the `slice(0,2)` but I don't want to inadvertently break things

https://github.com/Catrobat/Catroweb/blob/9bd66b3965ef0fb0476ba856db9de7bc6fbda975/templates/Default/base.html.twig#L2